pkg/config/config.go

```diff
@@ -879,7 +871,26 @@ func InitConfig(config Config) {
 
 	// Process agent
 	config.SetKnown("process_config.dd_agent_env")
```

Comment:
Worth noting that there is some existing confusion with `process_config` and `DD_PROCESS_AGENT_` prefix - we had some folks expecting `DD_PROCESS_CONFIG_` be supported. Wondering if binding both would be a way to go.

Reply:
That sounds reasonable. I can go ahead and do that